src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -96,6 +96,14 @@ int sd_dhcp_lease_get_mtu(sd_dhcp_lease *lease, uint16_t *mtu) {
         return 0;
 }
 
+int sd_dhcp_lease_get_dhcp_server(sd_dhcp_lease *lease, struct in_addr *addr) {
```

Comment:
We already have `sd_dhcp_lease_get_server_identifier()`.

Reply:
ahh I missed.